### ISSUE TITLE
Support for connector list restricted data usage (whitelisting)

### ISSUE THREAD

**Issue by trueg** [International-Data-Spaces-Association/DataspaceConnector#477]
It is currently possible to define a policy rule which grants permissions to a single connector:
```
{
    "@type" : "ids:Constraint",
    "ids:rightOperand" : {
      "@value" : "https://example.com",
      "@type" : "http://www.w3.org/2001/XMLSchema#anyURI"
    },
    "ids:leftOperand" : {
      "@id" : "idsc:SYSTEM"
    },
    "ids:operator" : {
      "@id" : "idsc:SAME_AS"
    }
  }
```

If, however, one wanted to grant access to a list of connectors one individual contract offer is required for each connector. This is not only rather cumbersome but poses a big problem if the list becomes big. At that point it gets rather hard to parse which contract offer to use when negotiating a new agreement. The consumer essentially has to parse each policy to figure out which grants access to them. This is not an ideal way and exposes another serious problem which I will address in another issue.

The ids model already provides the necessary entities to describe a list of connectors as follows using rdf collections:
```
{
    "@type" : "ids:Constraint",
    "ids:rightOperand" : {
      "@list": [ {
        "@value" : "https://example.com/1",
        "@type" : "http://www.w3.org/2001/XMLSchema#anyURI"
      }, {
        "@value" : "https://example.com/2",
        "@type" : "http://www.w3.org/2001/XMLSchema#anyURI"
      }  ]
     },
    "ids:leftOperand" : {
      "@id" : "idsc:SYSTEM"
    },
    "ids:operator" : {
      "@id" : "idsc:IN"
    }
  }
```

Definition from the ids model:
```
# Set operator
idsc:IN a ids:BinaryOperator;
    skos:broader odrl:isAnyOf, <https://postgis.net/docs/ST_Within.html> ;
    rdfs:label "in"@en ;
    rdfs:comment "Element-of operator. Allowed RightOperands are RDF collections('(A B C)')."@en ;
```

**Comment by juliapampus** [comment]
Waiting for Infomodel library update.

In the meantime: Use the possibility to add multiple contract offers to one resource. With this, you can add one contract that restricts the data usage to connector (and company) A, and another one that restricts the data usage to connector (and company) B.
